### PR TITLE
P18 CDR1 Display Conditions

### LINKED ISSUE #35: Convert P18 CDR to TopSky map with conditions
Remove P18 from UKSF.
Convert to TopSky Map
Create region as per map to infill CAS when not available.

### PR DESCRIPTION
**Pull request by luke11brown**
Fixes #35 

- Remove P18 from sectorfile between ADN and NATEB
- Create TopSky map to replace removed lines
- Create TopSky map polygon to fill in CAS blank when P18 not available

UKSF items to action:

- [x] P18 between ADN - NATEB to be separated from "AIRWAYS" defintion https://github.com/VATSIM-UK/UK-Sector-File/issues/3035
- [x] Affected Airspace bases require an asterisk anyway to display that it is a conditional route. https://github.com/VATSIM-UK/UK-Sector-File/issues/3033

### CODE REVIEW

File: UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt [hunk 1]
```diff
@@ -2666,4 +2667,128 @@ STYLE:solid:1
...
+COORD:N055.26.47.000:W001.42.33.000
+COORD:N056.43.26.000:W002.11.37.000
+COORD:N056.55.06.000:W002.19.52.000
+COORDPOLY:100
+ACTIVE:1
+//ACTIVE:0101:1231:1:0900:0000
+//ACTIVE:0101:1231:2345:0000:0429
+//ACTIVE:0101:1231:234:0800:0000
+//ACTIVE:0101:1231:5:1:0000:1459
+//ACTIVE:0101:1231:67:0000:2359
```
Comment (luke11brown) [inline UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt]: @hsugden Could you please double check the activation schedule I've set against the AIP. Thanks.

File: UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt [hunk 2]
```diff
@@ -2666,4 +2667,128 @@ STYLE:solid:1
...
+TEXT:N056.49.12.702:W001.54.21.335:FL75
+TEXT:N056.34.41.736:W001.58.55.465:FL115
+TEXT:N056.20.52.901:W001.54.38.318:FL135
+TEXT:N055.57.05.463:W001.44.50.553:FL155
+TEXT:N055.21.24.306:W001.35.21.156:FL155
+ACTIVE:0
+//ACTIVE:0101:1231:1:0000:0859
+//ACTIVE:0101:1231:2345:0430:0759
+//ACTIVE:0101:1231:5:1400:0000
+//ACTIVE:0101:1231:67:0000:2359
```
Comment (luke11brown) [inline UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt]: @hsugden Could you please double check the activation schedule I've set against the AIP. Thanks.

File: UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt [hunk 3]
```diff
@@ -2666,4 +2667,126 @@ STYLE:solid:1
...
+TEXT:N056.44.17.316:W002.03.12.612:FL75
+TEXT:N056.49.12.702:W001.54.21.335:FL75
+TEXT:N056.34.41.736:W001.58.55.465:FL115
+TEXT:N056.20.52.901:W001.54.38.318:FL135
+TEXT:N055.57.05.463:W001.44.50.553:FL155
+TEXT:N055.21.24.306:W001.35.21.156:FL155
+ACTIVE:0101:1231:1:0000:0859
+ACTIVE:0101:1231:2345:0430:0759
+ACTIVE:0101:1231:5:1400:0000
+ACTIVE:0101:1231:67:0000:2359
```
Comment (luke11brown) [inline UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt:2753]: @hsugden Could you please double check the activation schedule I've set against the AIP. Thanks.

File: UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt [hunk 4]
```diff
@@ -2666,4 +2667,126 @@ STYLE:solid:1
...
+COORD:N055.06.42.402:W001.49.20.883
+COORD:N055.26.47.000:W001.42.33.000
+COORD:N056.43.26.000:W002.11.37.000
+COORD:N056.55.06.000:W002.19.52.000
+COORDPOLY:100
+ACTIVE:0101:1231:1:0900:0000
+ACTIVE:0101:1231:2345:0000:0429
+ACTIVE:0101:1231:234:0800:0000
+ACTIVE:0101:1231:5:1:0000:1459
+ACTIVE:0101:1231:67:0000:2359
```
Comment (luke11brown) [inline UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt:2792]: @hsugden Could you please double check the activation schedule I've set against the AIP. Thanks.

File: UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt (hunk 3 above)
Comment (hsugden) [reply UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt:2753]: Pretty sure it's 0530-0900 local, so you might have to update the dates according to GMT/BST each year?

Comment (luke11brown) [reply UK-TopSky/Data/Plugin/TopSky/TopSkyMaps.txt]: 2b53e24b9748fd8a914b366bdb7f1d64b7a9cb4c

Added DST (valid now until next October) and May - Sep conditions

**Comment by luke11brown** [comment]
Now ready to merge, awaiting Sector File amendments

**Pull request merged**